helm/examples/efs/hpcc-efs/templates/hpcc-efs-server-sc.yaml

```diff
@@ -65,7 +65,8 @@ spec:
 kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
-  name: efs
+  name: {{ printf "%s" (include "hpcc-efs.fullname" $) }}
```

Comment:
to be consistent with our other examples, this should be something like "%s-sc"

Reply:
I make it like in azure case:  <dali|data|dll>-%s-sc